applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
<p>Absolutely a take it or leave it suggestion, but this might work better as an admonition, especially since it's more meta content. I'd also consider adding it in a different section, like the overview, as I'm not sure I'd look for this info under this heading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>